features/fetchlater.yml

```
@@ -0,0 +1,7 @@
+name: fetchLater
+description: The `fetchLater()` method requests a deferred fetch. After this method is called, the network request is queued by the browser, and is then invoked either when the document is destroyed, or after a certain time. When the request is eventually sent, the response is ignored by the browser.
```

Maybe this description is more clear?

```suggestion
description: The `fetchLater()` method requests a deferred fetch sent at an unknown time. The browser ensures a reliable time to send it, ideally when the document is unloaded. This API is useful for sending beacons, that is sending data to a server, without expecting a particular response.
```